tests/hashed-apikey-rls.test.ts

```
@@ -513,3 +564,88 @@ describe('RLS policies with hashed API keys (via Supabase SDK)', () => {
     expect(data).toEqual([])
   })
 })
+
+describe('Webhook and webhook_delivery RLS with API-key org scope precedence', () => {
```

ESLint

[error] `describe`s should begin with lowercase

(test/prefer-lowercase-title)